configure.py

```diff
@@ -1392,6 +1417,9 @@ def query_seastar_flags(seastar_pc_file, link_static_cxx=False):
         f.write('rule libdeflate.{mode}\n'.format(**locals()))
         f.write('    command = make -C libdeflate BUILD_DIR=../build/{mode}/libdeflate/ CFLAGS="{libdeflate_cflags}" CC={args.cc} ../build/{mode}/libdeflate//libdeflate.a\n'.format(**locals()))
         f.write('build build/{mode}/libdeflate/libdeflate.a: libdeflate.{mode}\n'.format(**locals()))
+        f.write('build build/{mode}/zstd/lib/libzstd.a: ninja\n'.format(**locals()))
+        f.write('   subdir = build/{mode}/zstd\n'.format(**locals()))
+        f.write('   target = libzstd.a\n'.format(**locals()))
 
```

Comment:
We should rename seastar_pool to submodule_pool and build all the submodules in that pool. Out of scope for this patchset, but would be nice to follow up.

Reply:
I added a commit in this PR, I think it won't hurt.